source/views/dictionary/list.js

```diff
@@ -105,11 +106,17 @@ export class DictionaryView extends React.Component {
   }
 
   performSearch = (text: string) => {
+    // Android clear button returns an object
+    if (!isString(text)) {
+      this.setState({results: terms})
+      return
+    }
```

Comment:
Thank you for writing `splitToArray`, `termToArray`, and `performSearch`, @hawkrives !